platform/javascript/export/export_plugin.cpp

```diff
@@ -142,12 +143,18 @@ void EditorExportPlatformJavaScript::_fix_html(Vector<uint8_t> &p_html, const Re
 		head_include += "<script type='application/javascript'>window.addEventListener('load', () => {if ('serviceWorker' in navigator) {navigator.serviceWorker.register('" +
 				p_name + ".service.worker.js');}});</script>\n";
 	}
-
+	else {
+		const String date = Time::get_singleton()->get_date_string_from_system();
+		const String time = Time::get_singleton()->get_time_string_from_system();
```

Comment:
You can use `get_datetime_string_from_system()` directly.

Might be worth setting `utc` to true too.